src/message_bus.zig

```diff
@@ -709,7 +709,18 @@ fn MessageBusType(comptime process_type: vsr.ProcessType) type {
                     return null;
                 }
 
-                const header: *const Header = @alignCast(mem.bytesAsValue(Header, data[0..@sizeOf(Header)]));
+                // If the message bus receives more than one message at a time, subsequent messages
+                // might not be aligned to 16. These messages would be copied over to a fresh
+                // `Message` anyway, fixing the alignment issue, but care must be taken when
+                // working with the header before that.
+                var header_aligned: Header = undefined;
+                const header: *const Header = // if (connection.recv_parsed % @alignOf(Header) == 0)
```

Comment:
I wonder if we should just copy it onto the stack header unconditionally... i.e. the cost of the branch even in the "happy case" might be more than the time added by the copy. Probably not measurable difference either way, though.

Reply:
Yeah, good call, let's keep it simple. If we are optimizing this, we really should do the "checksum unalinged data" approach, not much sense to implement half-optimisation.